docker/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # syntax=docker/dockerfile:1
 
-ARG PYTHON_VERSION=3.9
+ARG PYTHON_VERSION=3.10
```

Comment:
Since the security score of the Python3.9 container is much better than the one of 3.10 I would prefer to stay on 3.9 until they fixed this

Reply:
Okay-- wasn't sure why it was at 3.9 because the other docs said it was tested on 3.10 and known to be working, but I can change it back.

Reply:
It is indeed working (even when using the 3.9.* python-dev package on python 3.10), but the 3.9 image had only 25% of the security issues I got with 3.10. Since there are people which use this image in the cloud, security should be a high prio 😅

Reply:
Gotcha... I'm going back to 3.9....

Quick question-- I presume you have this running w/a gpu.  To run it on my machine w/nvidia support I needed to rely on a base image that supported cuda, namely nvidia/cuda...  I was trying to figure out-- where/how does this docker image get the gpu support from?